### ISSUE TITLE
Standard url category AI_ML_APPS not supported in provider - Error: expected url_categories.0 to be one of...

### ISSUE THREAD

**Issue by shmoeshmeps** [zscaler/terraform-provider-zia#250]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

terraform --version
Terraform v1.4.6
on linux_amd64
+ provider registry.terraform.io/zscaler/zia v2.5.5

### Affected Resource(s)

- zia_url_categories

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior
You should be able to plan or validate when a custom category uses the standard AI_ML_APPS category

### Actual Behavior
If you have a URL filter which uses the AI_MP_APPS URL category, you get an error when you `terraform plan` or `terraform validate` because it is not hardcoded as a standard URL CATEGORY in `validator.go`

`Error: expected url_categories.0 to be one of [ANY NONE OTHER_ADULT_MATERIAL...], got AI_ML_APPS`

### Steps to Reproduce
- Create a URL filter in the GUI which matches url category "AI and ML Applications"
- Import that resource
- Run terraform plan

### Important Factoids
- If I comment out the reference to this URL Category in my URL filter, the plan works - but then shows it's removing the category
- The AI_ML_APPS category is visible in terraform state

**Comment by willguibr** [comment]
@shmoeshmeps A new version of the provider  (v2.5.7) will be released by end of this week. The source code now includes the new URL category `AI_ML_APPS`.
Regards

**Comment by willguibr** [comment]
Hi @shmoeshmeps This issue has been addressed in the latest release v2.6.0. Please test and let us know if there's any problems.
Thank you
